creator-node/src/serviceRegistry.js

```diff
@@ -26,28 +26,27 @@ class ServiceRegistry {
     this.ipfsLatest = ipfsLatest
     this.blacklistManager = BlacklistManager
     this.monitoringQueue = new MonitoringQueue()
+    this.nodeConfig = config
 
-    // this.audiusLibs isn't initialized until
-    // `initServices` is called.
+    // below properties aren't initialized until 'initServices' is called
     this.audiusLibs = null
-    // this.snapbackSM isn't initialized until 'initServices'
     this.snapbackSM = null
   }
 
   /**
-   * Configure services, init libs.
+   * Configure all services, including libs init
    */
   async initServices () {
     // Initialize private IPFS gateway counters
     this.redis.set('ipfsGatewayReqs', 0)
     this.redis.set('ipfsStandaloneReqs', 0)
 
     await this.blacklistManager.init()
-    const audiusLibs = (config.get('isUserMetadataNode')) ? null : await initAudiusLibs()
-    this.libs = audiusLibs
-    // Conditionally initialize state machine
+
     if (!config.get('isUserMetadataNode')) {
-      this.snapbackSM = new SnapbackSM(this.libs)
+      this.audiusLibs = await initAudiusLibs()
```

Comment:
this is just a code refactor plus renames `this.libs` to `this.audiusLibs` (this is only consumed in `src/app.js`)